compiler/testData/ir/irText/stubs/builtinMap.fir.txt

```diff
@@ -18,7 +18,7 @@ FILE fqName:<root> fileName:/builtinMap.kt
             if: CONST Boolean type=kotlin.Boolean value=true
             then: CALL 'public final fun apply <T> (block: @[ExtensionFunctionType] @[ExtensionFunctionType] kotlin.Function1<T of kotlin.apply, kotlin.Unit>): T of kotlin.apply [inline] declared in kotlin' type=java.util.LinkedHashMap<K1 of <root>.plus?, V1 of <root>.plus?> origin=null
               <T>: java.util.LinkedHashMap<K1 of <root>.plus?, V1 of <root>.plus?>
-              $receiver: CONSTRUCTOR_CALL 'public constructor <init> (p0: kotlin.collections.Map<out K of java.util.LinkedHashMap?, out V of java.util.LinkedHashMap?>?) declared in java.util.LinkedHashMap' type=java.util.LinkedHashMap<K1 of <root>.plus?, V1 of <root>.plus?> origin=null
+              $receiver: CONSTRUCTOR_CALL 'public constructor <init> (p0: @[FlexibleNullability] kotlin.collections.Map<out K of java.util.LinkedHashMap?, out V of java.util.LinkedHashMap?>?) declared in java.util.LinkedHashMap' type=java.util.LinkedHashMap<K1 of <root>.plus?, V1 of <root>.plus?> origin=null
```

Comment:
Here, e.g., type parameters, `K`, `V`, 'K1`, and `V1`, need the same attribute, but not sure where it could be added/propagated...

Reply:
I'd say that type parameter based type should have this attribute if the referenced Java type parameter does not have not-null bound.

Reply:
At least `K` and `V` have that attribute for now. Not sure about `K1` and `V1`...